File: xwallpaper/src/main/java/vn/loitp/app/activity/customviews/textview/zoomtextview/ZoomTextViewActivity.java
package vn.loitp.app.activity.customviews.textview.zoomtextview;

import android.app.Activity;
import android.os.Bundle;

import vn.loitp.app.activity.customviews.textview.scrollnumber.lib.MultiScrollNumber;
import vn.loitp.app.base.BaseActivity;
import com.loitp.xwallpaper.R;

public class ZoomTextViewActivity extends BaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected Activity setActivity() {
        return this;
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_zoom_textview;
    }

}

File: app/src/main/java/vn/loitp/app/a/customviews/textview/zoom/ZoomTextViewActivity.kt
package vn.loitp.app.a.customviews.textview.zoom

import android.os.Bundle
import com.loitp.annotation.IsFullScreen
import com.loitp.annotation.LogTag
import com.loitp.core.base.BaseFontActivity
import com.loitp.core.utilities.LUIUtil
import kotlinx.android.synthetic.main.activity_text_view_zoom.*
import vn.loitp.R

@LogTag("ZoomTextViewActivity")
@IsFullScreen(false)
class ZoomTextViewActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_text_view_zoom
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setupViews()
    }

    private fun setupViews() {
        lActionBar.apply {
            LUIUtil.setSafeOnClickListenerElastic(
                view = this.ivIconLeft,
                runnable = {
                    onBaseBackPressed()
                }
            )
            this.ivIconRight?.setImageResource(R.color.transparent)
            this.tvTitle?.text = ZoomTextViewActivity::class.java.simpleName
        }
    }
}
